src/main/resources/application.properties

```diff
@@ -1 +1,2 @@
-# camel.springboot.main-run-controller=true to run the app continuously
\ No newline at end of file
+# camel.springboot.main-run-controller=true to run the app continuously
+logging.level.org.apache.commons.httpclient.HttpMethodBase = ERROR
```

Comment:
Yes, I believe that's what's used for the actual Sierra API call. This was added to prevent cookie error messages on the API calls that I thought we could safely ignore.